src/www/ui_tests/api/Controllers/JobControllerTest.php

```diff
@@ -154,6 +179,12 @@ public function testGetJobs()
     $request = new Request("GET", new Uri("HTTP", "localhost"),
       $requestHeaders, [], [], $body);
     $response = new ResponseHelper();
+    $userId = 2;
+    $user = $this->getUsers([$userId]);
+    $user[0]["default_group"] = "fossy";
+    $this->restHelper->shouldReceive('getUserId')->andReturn($userId);
+    $this->dbHelper->shouldReceive('getUsers')->withArgs([$userId])
+      ->andReturn($user);
```

Comment:
These might change as well.

```suggestion
    $user = $this->getUsers([$userId]);
    $this->restHelper->shouldReceive('getUserId')->andReturn($userId);
```

Reply:
resloved